Documentation/ceph-csi-snapshot.md

```diff
@@ -4,23 +4,19 @@ weight: 3250
 indent: true
 ---
 
-## RBD Snapshots
-
-### Prerequisites
+## Prerequisites
 
-1. Requires Kubernetes v1.17+ which supports snapshot beta.
+- Requires Kubernetes v1.17+ which supports snapshot beta.
 
-2. Install the new snapshot controller and snapshot beta CRD. More info can be found
+- Install the new snapshot controller and snapshot beta CRD. More info can be found
 [here](https://github.com/kubernetes-csi/external-snapshotter/tree/v2.1.1#usage)
 
 Note: If the Kubernetes distributor you are using does not supports the snapshot beta,
 still you can use the Alpha snapshots. refer to
 [snapshot](https://github.com/rook/rook/blob/release-1.3/Documentation/ceph-csi-drivers.md#rbd-snapshots)
 on how to use snapshots.
 
-### SnapshotClass
-
-You need to create the `SnapshotClass`. The purpose of a `SnapshotClass` is
+- Create the `SnapshotClass`. The purpose of a `SnapshotClass` is
```

Comment:
the snapshot class cannot be `Prerequisites` isn't it? as we are talking about how to create snapshotclass below.

Reply:
Adjusted this a bit.. Ptal.